src/screen/MyIdentityScreen/index.tsx

```diff
@@ -1,38 +1,277 @@
+            )}
+            accessoryRight={() => (
+              <Text selectable category="label" numberOfLines={1} style={styles.address} ellipsizeMode="middle">
+                {address}
+              </Text>
+            )}
+          />
+          {identity?.hasIdentity ? (
+            <>
+              <ListItem
+                disabled
+                title="Display"
+                accessoryLeft={(iconProps: IconProps) => <Icon {...iconProps} name="person-outline" />}
+                accessoryRight={() => (
+                  <Text selectable category="label" numberOfLines={1} ellipsizeMode="middle">
+                    {identity?.display}
+                  </Text>
+                )}
+              />
+              <MenuGroup title=" Identity detail" accessoryLeft={MoreIcon}>
+                <MenuItem
+                  title="Legal"
+                  accessoryLeft={(props) => <Icon {...props} name="award-outline" />}
+                  accessoryRight={() => (
+                    <Text selectable category="label">
+                      {identity.registration?.legal || 'Unset'}
+                    </Text>
+                  )}
+                />
+                <MenuItem
+                  title="Email"
+                  accessoryLeft={(props) => <Icon {...props} name="email-outline" />}
+                  accessoryRight={() => (
+                    <Text selectable category="label">
+                      {identity.registration?.email || 'Unset'}
+                    </Text>
+                  )}
+                />
+                <MenuItem
+                  title="Twitter"
+                  accessoryLeft={(props) => <Icon {...props} name="twitter-outline" />}
+                  accessoryRight={() => (
+                    <Text selectable category="label">
+                      {identity.registration?.twitter || 'Unset'}
+                    </Text>
+                  )}
+                />
+                <MenuItem
+                  title="Riot"
+                  accessoryLeft={(props) => <Icon {...props} name="message-square-outline" />}
+                  accessoryRight={() => (
+                    <Text selectable category="label">
+                      {identity.registration?.riot || 'Unset'}
+                    </Text>
+                  )}
+                />
+                <MenuItem
+                  title="Web"
+                  accessoryLeft={(props) => <Icon {...props} name="browser-outline" />}
+                  accessoryRight={() => (
+                    <Text selectable category="label">
+                      {identity.registration?.web || 'Unset'}
+                    </Text>
+                  )}
+                />
+              </MenuGroup>
+            </>
+          ) : null}
+
+          <Padder scale={1} />
+          <Button onPress={() => modalRef.current?.open()} status="basic">
+            {identity?.hasIdentity ? 'Complete Identity' : 'Set Identity'}
```

Comment:
I think the word complete may confuse the users, I'm still not completely sure I understand the idea behind "Complete". How about **Update Identity** or **Edit Identity**?